src/Simplex/Chat/Controller.hs

```diff
@@ -420,7 +458,17 @@ data ChatCommand
   deriving (Show)
 
 data ChatResponse
-  = CRActiveUser {user :: User}
+  = CRRemoteHostCreated {ident :: RemoteHostId, oobData :: Text}
```

Comment:
```suggestion
  = CRRemoteHostCreated {remoteHostId :: RemoteHostId, oobData :: Text}
```

Reply:
TBD. The only use for this right now is CA fingerprint.